src/documents/ImageReordererAPI.md

```diff
@@ -0,0 +1,488 @@
+                * `up`
+                * `down`
+                * `right`
+                * `left`
+            </td>
+            <td>
+                <pre>
+                    <code>
+fluid.reorderer.defaultKeysets = [{
+    modifier : function (evt) {
+        return evt.ctrlKey;
+    },
+    up : fluid.reorderer.keys.UP,
+    down : fluid.reorderer.keys.DOWN,
+    right : fluid.reorderer.keys.RIGHT,
+    left : fluid.reorderer.keys.LEFT
+},
+{
+    modifier : function (evt) {
+        return evt.ctrlKey;
+    },
+    up : fluid.reorderer.keys.i,
+    down : fluid.reorderer.keys.m,
+    right : fluid.reorderer.keys.k,
+    left : fluid.reorderer.keys.j
+}];
+                    </code>
+                </pre>
+            </td>
+        </tr>
+
+        <tr>
+            <td>selectablesTabindex</td>
+            <td>Normally injected automatically from the layoutHandler</td>
+            <td>String [IoC expression](IoCReferences.md)</td>
+            <td>
+                <pre>
+                    <code>
+selectablesTabindex: "{that}.layoutHandler.options.selectablesTabindex"
+                    </code>
+                </pre>
+            </td>
+        </tr>
+
+        <tr>
+            <td>avatarCreator</td>
+            <td>a function that returns a valid DOM node to be used as the dragging avatar</td>
+            <td></td>
+            <td>
+
+The item being dragged will be cloned</td>
+        </tr>
+
+        <tr>
+            <td>disableWrap</td>
+            <td>This option is used to disable wrapping of elements within the container.</td>
+            <td>boolean</td>
+            <td>false</td>
+        </tr>
+
+        <tr>
+            <td>mergePolicy</td>
+            <td>an object describing how user options should be merged in with defaults
+For information on options merging, see [Options Merging](OptionsMerging.md)</td>
+            <td></td>
+            <td>
+                <pre>
+                    <code>
+mergePolicy: {
+    keysets: "replace",
+    "selectors.selectables":
+        "selectors.movables",
+    "selectors.dropTargets":
+        "selectors.movables"
+}
+                    </code>
+                </pre>
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+### Image Reorderer Options
+
+<table>
+    <thead>
+        <tr>
+            <th>Name</th>
+            <th>Description</th>
+            <th>Values</th>
+            <th>Default</th>
+        </tr>
+    </thead>
+
+    <tbody>
+        <tr>
+            <td><strong>Deprecated as of 1.1.2:</strong>
+afterMoveCallbackUrl</span></td>
+            <td>If an URL is provided with this option, the current state of the component model will be sent to that URL after a move is carried out using a default afterMove event handler. See [Deprecated - afterMoveCallback](http://wiki.fluidproject.org/display/Infusion13/Deprecated+-+afterMoveCallback) for info. </td>
```

Comment:
Since this is deprecated, let's remove the reference to the docs on the wiki altogether.